docs/contributing/guidelines-for-writing-documentation.md

```diff
@@ -2,7 +2,25 @@
 
 * Documentation is distributed with the source code of Shopsys Framework inside [docs](https://github.com/shopsys/shopsys/tree/7.3/docs/) directory in the project root.
 * Documentation is written in [Markdown format](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet).
+* Output HTML is rendered from the `*.md` files using [mkdocs](https://www.mkdocs.org/) and hosted on [readthedocs](https://readthedocs.org/).
+    * The docs live at [https://docs.shopsys.com](https://docs.shopsys.com).
+    * When using Docker, you can see the rendered docs locally on `http://127.0.0.1:1300`.
+        * See `mkdocs` container defined in `docker-compose.yml`.
+    * You can even access the rendered docs under `/documentation/` sub-folder for any branch that is built on our Jenkins CI server.
+    * The main [`mkdocs.yml`](https://github.com/shopsys/shopsys/blob/7.3/mkdocs.yml) configuration file is located in the monorepo root.
 * All documentation files should be named same as first heading (in lowercase, non-alphanumeric characters replaced by dash).
 * References to project files and classes have to be absolute links to the GitHub in proper version - eg. `[app/config/config.yml](https://github.com/shopsys/shopsys/blob/7.3/project-base/app/config/config.yml)`.
+* References among the docs files must be relative.
 * All words in a title except conjunctions (and, or, but...), articles (a, an, the), and short prepositions (in, to, of...) should be capitalized. Other headings should not be capitalized.
-
+* Each sub-folder in the `docs` folder should contain `index.md` file with links to all the other articles in the sub-folder as well as `navigation.yml` file with the menu configuration that should be consistent with the `index.md`.
+* If using a list in the text, you need to add an empty line above the list definition, otherwise it is not rendered properly.
+* Rendering of multi-line code-blocks in the lists does not work properly so if you need to add such thing in your article, consider using headlines instead of bullet points in your text flow, or just do not indent the code-blocks at all in your list.
+* Enabled `mkdocs` plugins and extensions:
+    * [`toc`](https://python-markdown.github.io/extensions/toc/)
+        * You can use `[TOC]` tag to generate a table of contents for your article.
+    * [`admonition`](https://python-markdown.github.io/extensions/admonition/)
+        * You can add nicely rendered blocks with notes, warnings, tips, etc. using `!!! note/warning/...` syntax.
```

Comment:
I wouáld recommend to add list of supported types so we know what we should prepare when coding the design for docs site.

Reply:
Well, what do you mean by "supported"? As the linked docs state, we can use all the rST-style types - see http://docutils.sourceforge.net/docs/ref/rst/directives.html#specific-admonitions. We can discuss the styling but I think it should not block this PR :slightly_smiling_face:

Reply:
oh, so I can use any from tis list: "attention", "caution", "danger", "error", "hint", "important", "note", "tip", "warning", "admonition"

If it is so, I would recommend putting there a list or link that you send me. Or is this some general developers knowledge that there is a list like that?

Reply:
it is linked from the extension docs that are linked in our docs, however, I have added a link to the rSt docs for clarity in fixup commit - 2367bc8fc

Reply:
nice one! thank you